color/d2v-color-common/src/main/kotlin/io/data2viz/color/Color.kt

```diff
@@ -19,15 +21,59 @@ interface Color : ColorOrGradient {
      * where the number 1 corresponds to 100% (full opacity).
      */
     val rgba:String
+
     val r:Int
     val g:Int
     val b:Int
     val alpha:Double
     val rgbHex:String
+
+    /**
+     * The relative brightness normalized to 0% for darkest black and 100% for lightest white.
+     * According to the WCAG definition: https://www.w3.org/TR/2008/REC-WCAG20-20081211/#relativeluminancedef
+     */
+    fun luminance():Percent
+
+    /**
+     * The contrast ratio between 2 colors.
+     * A minimum contrast of 4.5:1 is recommended to ensure that text is still readable against a background color.
+     * According to the WCAG definition: https://www.w3.org/TR/2008/REC-WCAG20-20081211/#contrast-ratiodef
+     */
+    fun contrast(other:Color):Double
+
+    /**
+     * Return `true` if 2 colors have a contrast of 4.5 or more.
+     * According to the WCAG definition: https://www.w3.org/TR/WCAG20-TECHS/G18.html
+     */
+    fun isContrastOK(other:Color):Boolean
```

Comment:
The referenced article is very precise about its target: we are talking about the contrast needed between a text and its background. The name of the function should represent this purpose.

> this technique describes the minimum contrast ratio for text that is less than 18 point (if not bold) and less than 14 point (if bold). For Success Criterion 1.4.5, this technique relaxes the 7:1 contrast ratio requirement for text that is at least 18 point (if not bold) or at least 14 point (if bold).

Reply:
I remove it, but "luminance" and "contrast" will remains and will certainly be used in charts.kt (in a more precise way including font size etc...)